pkgs/tools/misc/go.rice/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "rice";
+  version = "1.0.0";
+
+  name = "rice";
+  src = fetchFromGitHub {
+    owner = "GeertJohan";
+    repo = "go.rice";
+    rev = "v${version}";
+    sha256 = "0m1pkqnx9glf3mlx5jdaby9yxccbl02jpjgpi4m7x1hb4s2gn6vx";
+  };
+
+  vendorSha256 = "0cb5phyl2zm1xnkhvisv0lzgknsi93yzmpayg30w7jc6z4icwnw7";
+
+  subPackages = [ "." "rice" ];
```

Comment:
You can probably remove the ".". Also, it may work just fine with no subPackages at all.

Reply:
https://github.com/GeertJohan/go.rice#installation

Unless I misinterpreted the regex that consumes `subPackages`, I think this is necessary.

Reply:
EDIT: This is too much black magic vodoo for my taste...
```
 156   │           pushd "$NIX_BUILD_TOP/go/src" >/dev/null
 157   │           find "$goPackagePath" -type f -name \*$type.go -exec dirname {} \; | grep -v "/vendor/" | sort | uni
       │ q
 158   │           popd >/dev/null
```

Reply:
`find . -type f -name \*$type.go -exec dirname {} \; | grep -v "/vendor/" | sort --unique` would also match: https://github.com/GeertJohan/go.rice/tree/master/example We don't want that built.